gobblin-service/src/main/java/org/apache/gobblin/service/modules/template/FlowTemplate.java

```diff
@@ -63,9 +63,9 @@
    * @param userConfig User supplied Config
    * @param inputDescriptor input {@link DatasetDescriptor}
    * @param outputDescriptor output {@link DatasetDescriptor}
-   * @return true if the {@link FlowTemplate} is resolvable
+   * @return any exception that occurred when resolving or null if the {@link FlowTemplate} is resolvable
```

Comment:
Changed to be void function and handle the exception outside (there is not really any point in it being boolean since it would always return true or throw exception).